infrastructure/web/controllers/get-users.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function getUsersList({ listUsers }) {
+  return async function usersList(req, res) {
```

Comment:
usersList -> getUsers for consistency ("get" is the name of the method and "users" is the route)